src/main/java/frc/robot/subsystems/shooter/commands/Shoot.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.shooter.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.shooter.Shooter;
+
+/**
+ * This command keeps the shooter in the specified velocity.
+ * This command does <b>NOT</b> change velocity of the shooter.
+ * The command should come in conjunction with a button, and not to start/stop it after a specified time.
+ * If you wish to accelerate the shooter to a specified velocity, please look at {@link SpeedUp}.
```

Comment:
Why not just use `SpeedUp`?

Reply:
speed up stops when the shooter reach to the desired velocity, this command just keep the shooter at the same speed.
I did it this way so we know whether we can move the balls from the conveyor later, when we integrate with other subsystems

Reply:
Also, where did I write that "this command doesn't check whether the shooter "is ready"."

Reply:
Oh, I got you, I'll fix the documentation.

Reply:
You didn't write so. But your code doesn't include that functionality :-).
However, I wasn't speaking about the documentation, but about the code itself. I'll continue on the other thread you and Adam are talking in below.